README.md

```diff
@@ -1,38 +1,71 @@
 
 My app seeks to resolve these issues by allowing users to find these lineups in a single place while also giving them the ability to save fests to which they're going. In order to ease fest day planning, users can plan which artists they want to see at each of those fests with the app as well.
 
-## Setup for Running Locally
-Requirements: [NPM](https://www.npmjs.com/get-npm) | [JSON-Server](https://www.npmjs.com/package/json-server)
+## Installation
+
+Requirements: [Yarn](https://yarnpkg.com/) _or_ [NPM](https://www.npmjs.com/get-npm)
 
 1. Clone this repository
-1. `cd` into the directory it creates
-1. In the `api` directory, create a copy of the `database.json.example` and remove the .example extension.
-1. From the `api` directory, run `json-server -p 5002 -w database.json`.
-1. Run `npm install` and wait for all dependencies to be installed
-1. From the project root directory, run `npm start`.
-1. Open [http://localhost:3000](http://localhost:3000) to view it in the browser in development mode.
+   ```sh
+   git clone https://github.com/manilabui/there.git
+   ```
+1. Navigate to the project directory
+   ```sh
+   cd there
+   ```
+1. Install dependencies
+   ```sh
+   yarn
+   # or if you prefer npm
+   npm install
+   ```
+1. Create the local database
+   ```sh
+   yarn create-db
+   # or
+   npm run create-db
+   ```
+1. Run a local instance of the database
+   ```sh
+   yarn serve
+   # or
+   npm run serve
+   ```
+1. Start your local development server of the React application.
+   ```sh
+   yarn start
+   # or
+   npm run start
+   ```
+1. If your browser doesnt open and navigate to the app automatically, navigate to [http://localhost:3000](http://localhost:3000) to view it in the browser in development mode.
 
 ---
 
-###### Any user, including those without accounts, may look at the list of available fests as well as their corresponding lineups and schedules.
+##### Any user, including those without accounts, may look at the list of available fests as well as their corresponding lineups and schedules.
+
 ![](src/assets/there_loggedOutUser.gif)
 
-###### Logged-in users can see a timeline of their upcoming events, and they may also either confirm or express interest (checkmark icon vs star icon) of their attendance to a set.
+##### Logged-in users can see a timeline of their upcoming events, and they may also either confirm or express interest (checkmark icon vs star icon) of their attendance to a set.
+
 ![](src/assets/there_loggedInUser.gif)
 
 ---
 
-#### Technology Stack
-- Mockup: Figma (Check out the mockup [here](https://www.figma.com/file/9VNwMnHPQYdneR0sDGsgWy/T.H.E.R.E.?node-id=4%3A2))
-- JS: React with Hooks
-- Styles: Tachyons, CSS
-- Libraries: Lodash, Moment
-- Graphics: Logo + some custom icons designed by me in Figma (Others were Material icons, also from Figma.)
-- DB: JSON Server
-- Version Control: Git, GitHub
+## Technologies Used
```

Comment:
Changed the Heading level of this to match the others. I think its the right stylistic call. I'm unsure about how I italicized the bullet keys, but I think the Key/Values need to stand out from each other.